CONTRIBUTING.md

```diff
@@ -1237,7 +1237,6 @@ Note the `y_axis_format` is defined under various section for some charts.
 | `timed_refresh_immune_slices`   | _N/A_ |       |
 | `toggle_polygons`               | _N/A_ |       |
 | `transpose_pivot`               | _N/A_ |       |
-| `treemap_ratio`                 | _N/A_ |       |
```

Comment:
@kristw I'm assuming these should get yanked when things are moved, but correct me if I'm wrong.